### PR TITLE
Added more info for pool overview data

### PR DESCRIPTION
**Pull request by Mi-Lan**

### CODE REVIEW

File: app/src/components/market/sections/market_pooling/market_pool_liquidity.tsx [hunk 1]
```diff
@@ -69,21 +69,36 @@ const WarningMessageStyled = styled(WarningMessage)`
...
 `
 
 const UserDataTitleValue = styled(TitleValue)`
   width: calc(50% - 16px);
 `
+const UserDataRow = styled.div`
+  display: flex;
+  justify-content: space-between;
+  &:first-child {
+    margin-bottom: 10px;
```
Comment (pimato) [inline app/src/components/market/sections/market_pooling/market_pool_liquidity.tsx:85]: should be 12px

**Review by pimato — CHANGES_REQUESTED** [review]
do we have the if profit show green balance?
Bildschirmfoto 2020-10-28 um 14 50 48

I don`t see this in the code.

**Comment by Mi-Lan** [comment]
> do we have the if profit show green balance?
> Bildschirmfoto 2020-10-28 um 14 50 48
> 
> I don`t see this in the code.

yes actually 
Screenshot 2020-10-28 at 15 58 36
this part right here state prop is for color valuestate.success that is from userDataTitleValue component and in the value props the first part is dynamic if there are gains...

**Review by pimato — APPROVED** [review]

File: app/src/components/market/sections/market_pooling/market_pool_liquidity.tsx [hunk 2]
```diff
@@ -301,17 +316,32 @@ const MarketPoolLiquidityWrapper: React.FC<Props> = (props: Props) => {
...
-        <UserDataTitleValue
-          state={userEarnings.gt(0) ? ValueStates.success : undefined}
-          title={'Your Earnings'}
-          value={`${userEarnings.gt(0) ? '+' : ''}${formatNumber(formatBigNumber(userEarnings, collateral.decimals))} ${
-            collateral.symbol
-          }`}
-        />
+        <UserDataRow>
+          <UserDataTitleValue
+            title={'Your Liquidity'}
```
Comment (liaojikunwork) [inline app/src/components/market/sections/market_pooling/market_pool_liquidity.tsx:321]: @Mi-Lan , could you remove `{` and `}` when you send strings as props.

File: app/src/components/market/sections/market_pooling/market_pool_liquidity.tsx [hunk 3]
```diff
@@ -301,17 +316,32 @@ const MarketPoolLiquidityWrapper: React.FC<Props> = (props: Props) => {
...
+            value={`${formatNumber(formatBigNumber(totalUserLiquidity, collateral.decimals))} ${collateral.symbol}`}
+          />
+          <UserDataTitleValue
+            title={'Total Pool Tokens'}
```
Comment (liaojikunwork) [inline app/src/components/market/sections/market_pooling/market_pool_liquidity.tsx:325]: Same here

File: app/src/components/market/sections/market_pooling/market_pool_liquidity.tsx [hunk 4]
```diff
@@ -301,17 +316,32 @@ const MarketPoolLiquidityWrapper: React.FC<Props> = (props: Props) => {
...
+            value={`${formatBigNumber(totalPoolShares, collateral.decimals)} ${collateral.symbol}`}
+          />
+        </UserDataRow>
+        <UserDataRow>
+          <UserDataTitleValue
+            state={userEarnings.gt(0) ? ValueStates.success : undefined}
+            title={'Your Earnings'}
```
Comment (liaojikunwork) [inline app/src/components/market/sections/market_pooling/market_pool_liquidity.tsx:332]: Here

File: app/src/components/market/sections/market_pooling/market_pool_liquidity.tsx [hunk 5]
```diff
@@ -301,17 +316,32 @@ const MarketPoolLiquidityWrapper: React.FC<Props> = (props: Props) => {
...
+            value={`${userEarnings.gt(0) ? '+' : ''}${formatNumber(
+              formatBigNumber(userEarnings, collateral.decimals),
+            )} ${collateral.symbol}`}
+          />
+          <UserDataTitleValue
+            state={totalEarnings.gt(0) ? ValueStates.success : undefined}
+            title={'Total Earnings'}
```
Comment (liaojikunwork) [inline app/src/components/market/sections/market_pooling/market_pool_liquidity.tsx:339]: here

**Review by liaojikunwork — APPROVED** [review]

**Pull request merged**